aggregator_api/src/tests.rs

```diff
@@ -82,7 +82,7 @@ async fn get_config() {
             .await,
         Status::Ok,
         concat!(
-            r#"{"dap_url":"https://dap.url/","role":"Either","vdafs":"#,
+            r#"{"protocol":"DAP-05","dap_url":"https://dap.url/","role":"Either","vdafs":"#,
```

Comment:
As elsewhere, we should advertise DAP-07
```suggestion
            r#"{"protocol":"DAP-07","dap_url":"https://dap.url/","role":"Either","vdafs":"#,
```